app/models/group_order_article.rb

```diff
@@ -128,15 +128,18 @@ def calculate_result(total = nil)
       order_quantities = GroupOrderArticleQuantity.where(group_order_article_id: order_article.group_order_article_ids).order('created_on')
       logger.debug "GroupOrderArticleQuantity records found: #{order_quantities.size}"
 
+      first_order_first_serve = (FoodsoftConfig[:distribution_strategy] == FoodsoftConfig::DistributionStrategy::FIRST_ORDER_FIRST_SERVE)
+
       # Determine quantities to be ordered...
       order_quantities.each do |goaq|
-        q = [goaq.quantity, total - total_quantity].min
+        q = goaq.quantity
```

Comment:
i'm not 100% happy with this piece of code, but it should be ok for now and can be adopted when we add additional strategies later.